src/figuresClasses.ts

```diff
@@ -1,11 +1,119 @@
-export interface Figure {}
+type ShapeType = `triangle` | `circle` | `rectangle`;
+type ColorType = `red` | `green` | `blue`;
 
-export class Triangle implements Figure {}
+function checkSidesLength(...sides: number[]): boolean {
```

Comment:
Why?
u can do it without the function.
I think this is redundant.
a triangle always has 3 sides, the `||` operator will enough